v2.1/explain.md

```diff
@@ -80,19 +78,14 @@ useful to find out which indexes and index key ranges are used by a query:
 (5 rows)
 ~~~
 
-The first column shows the tree structure of the query plan; a set of properties
-is displayed for each node in the tree. Most importantly, for scans, you can see
-the index that is scanned (`primary` in this case) and what key ranges of the
-index you are scanning (in this case, a full table scan). For more
-information on indexes and key ranges, see the
-[example](#find-the-indexes-and-key-ranges-a-query-uses) below.
+The first column shows the tree structure of the query plan; a set of properties is displayed for each node in the tree. Most importantly, for scans, you can see the index that is scanned (`primary` in this case) and what key ranges of the index you are scanning (in this case, a full table scan). For more information on indexes and key ranges, see the [example](#find-the-indexes-and-key-ranges-a-query-uses) below.
```

Comment:
Is there a content change in this para?  In general I would prefer formatting PRs be separate from content updates.  Rewrapping the para makes it difficult to see if content was changed.

No need to change this for this PR.  Warrants a team discussion to see if it's worth caring about.  (I sometimes make formatting changes too in a content PR, so maybe there's a happy medium to be found.